tap_urban_airship/schemas/lists.json

```diff
@@ -21,26 +21,26 @@
                 "type": "string"
               },
               "value": {
-                "type": "string"
+                "type": ["string", "null"]
               }
             }
           }
         }
       ]
     },
     "created": {
-      "type": "string",
+      "type": ["string", "null"]
```

Comment:
This is invalid JSON. This line is missing a comma.

How did you test your change?